calisphere/cache_retry.py

```diff
@@ -18,6 +18,9 @@
 
 requests.packages.urllib3.disable_warnings()
 
+from aws_xray_sdk.core import patch
+patch(('requests',))
```

Comment:
should make this conditional on `settings`?  Otherwise, the manage.py command to generate the sitemaps does not work